jupyter_notebook_config.py

```diff
@@ -76,3 +78,49 @@
             sys.exit(155)
     else:
         print('Tags variable is not an array.', file=sys.stderr)
+
+def saveFile(file_path, token):
+    """
+    Construct a requests POST call with args and kwargs and process the
+    results.
+    Args:
+        file_path: The relative path to the file from the datadir, including filename and extension
+        token: keboola storage api token
+    Returns:
+        body: Response body parsed from json.
+    Raises:
+        requests.HTTPError: If the API request fails.
+    """
+
+    url = 'http://data-loader-api/data-loader-api/save'
+    headers = {'X-StorageApi-Token': token, 'User-Agent': 'Keboola Sandbox Autosave Request'}
+    payload = {}
+    payload['file'] = {'source': file_path, 'tags': ['autosave']}
+
+    r = requests.post(url, json=payload, headers=headers, timeout=240)
```

Comment:
why 240?

Reply:
because of the 3min delay on errors

Reply:
ic!, please add a comment and a link to jira in that matter to the code, otherwise we'll be scratching our heads in a month :)

Reply:
good point! :-)